src/cmn/conversation.py

```diff
@@ -1,18 +1,24 @@
-import os
 import csv
 
 from cmn.message import Message
 
 
-class Conversation(Object):
-    def __init__(self, id, messages, participants):
+class Conversation:
+    def __init__(self, id):
         self.id = id
-        self.messages = messages
-        self.participants = participants
+        self.messages = []
+        self.participants = set()
+        self.conv_size = 0
 
     @staticmethod
     def loader(path):
-        if path.endswith(".csv"): return Conversation.csv_loader(path)
+        if path.endswith(".csv"):
+            return Conversation.csv_loader(path)
+
+    def add_message(self, message_content, author_involved, size):
```

Comment:
@arun-esh please remove this function as it does not prep. or post. processing. you can easily add the messages to the conversation object directly